podcasts/AuthenticationHelper.swift

```diff
@@ -30,16 +30,19 @@ class AuthenticationHelper {
 
     // MARK: Password
 
-    static func validateLogin(username: String, password: String, scope: AuthenticationScope) async throws -> AuthenticationResponse {
+    @discardableResult
+    static func validateLogin(username: String, password: String, scope: AuthenticationScope = .mobile) async throws -> AuthenticationResponse {
         let response = try await ApiServerHandler.shared.validateLogin(username: username, password: password, scope: scope.rawValue)
-        handleSuccessfulSignIn(response, .password)
+        SyncManager.clearTokensFromKeyChain()
+        ServerSettings.setSyncingEmail(email: username)
```

Comment:
Repeating saving the email here as a safety measure for Prod since the response won't have an email item yet.